app/services/people/import_from_nomis.rb

```diff
@@ -12,43 +12,56 @@ class ImportFromNomis
       prison_number: :prison_number,
     }.freeze
 
-    def initialize(prison_number)
-      @prison_number = prison_number
+    def initialize(prison_numbers)
+      @prison_numbers = prison_numbers
     end
 
     def call
-      person = Person.find_or_initialize_by(prison_number: @prison_number)
-      person.assign_attributes(person_attributes)
-      person.save!
+      @prison_numbers.each do |prison_number|
+        person = Person.find_or_initialize_by(prison_number: prison_number)
+        nomis_person = nomis_person(prison_number)
+
+        next unless nomis_person
```

Comment:
we do not need this if we filter the prison numbers earlier, before the each, something like:
```
@prison_numbers.select do
  people_from_nomis.find { |nomis_person| nomis_person[:prison_number] == prison_number }.any?
end
```